### PR TITLE
Update pen-tests.vue to add IncludeSec

### PR DESCRIPTION
**Pull request by includesec-erik**
IncludeSec's team has been performing SOC2 related testing since 2011. We have performed thousands of assessments for hundreds of clients in the US. Reach out to info [at sign] includesecurity.com if you need to get in contact with us. Thanks to Warren and Rhosys team for maintaining soc2.fyi!

### CODE REVIEW

File: src/components/pen-tests.vue [hunk 1]
```diff
@@ -117,6 +117,29 @@
...
+                <small class="d-flex justify-content-center flex-column">
+                  <span><i class="fa-solid fa-check text-success" /> <a href="https://owasp.org/www-project-application-security-verification-standard/" target="_blank" rel="noopener">OWASP ASVS</a></span>
+                  <span><i class="fa-solid fa-check text-success" /> Custom tailored methodologies</span>
+                </small>
+              </div>
+            </td>
+            <td><success /></td>
+            <td><success /></td>
+            <td class="text-success">Retests included and can be used within 12 months</td>
+            <td class="text-success">All senior team and thorough</td>
```
Comment (wparad) [inline src/components/pen-tests.vue]:
```suggestion
            <td class="text-success">Free retests within 12 months</td>
```

File: src/components/pen-tests.vue [hunk 2]
```diff
@@ -117,6 +117,29 @@
...
+                <span class="d-flex align-items-center justify-content-center">
+                  $10k+
+                </span>
+              </div>
+            </td>
+            <td>
+              <div class="d-flex justify-content-center align-items-center">
+                <small class="d-flex justify-content-center flex-column">
+                  <span><i class="fa-solid fa-check text-success" /> <a href="https://owasp.org/www-project-application-security-verification-standard/" target="_blank" rel="noopener">OWASP ASVS</a></span>
+                  <span><i class="fa-solid fa-check text-success" /> Custom tailored methodologies</span>
```
Comment (wparad) [inline src/components/pen-tests.vue]:
```suggestion
```

**Review by wparad — APPROVED** [review]

**Comment by wparad** [comment]
Thanks! merging this in.

File: src/components/pen-tests.vue [hunk 3]
```diff
@@ -131,14 +131,12 @@
...
                   <span><i class="fa-solid fa-check text-success" /> <a href="https://owasp.org/www-project-application-security-verification-standard/" target="_blank" rel="noopener">OWASP ASVS</a></span>
-                  <span><i class="fa-solid fa-check text-success" /> Custom tailored methodologies</span>
                 </small>
               </div>
             </td>
             <td><success /></td>
             <td><success /></td>
-            <td class="text-success">Retests included and can be used within 12 months</td>
-            <td class="text-success">All senior team and thorough</td>
+            <td class="text-success">Free retests within 12 months</td>
```
Comment (includesec-erik) [inline src/components/pen-tests.vue:139]: @wparad The retests are included in the original pricing, I'm not sure saying they're free is the right message. I don't want anybody reading this to get the wrong impression and send you a correction being upset that they aren't free!

Comment (wparad) [reply src/components/pen-tests.vue:139]: Honestly I'm not really sure what the difference is here, I just wanted to keep the language consistent with the other places we talk about this.

Realistically "free", always means "priced in", so whether or not it is extra or not isn't necessarily relevant. If anything I'm most concerned about the language "can be used within 12 months", that part felt the most not-obvious while reading. My expectation is that everyone understands what free means here, are you saying that you explicitly have a line item in the contract that says "retests - $K" in the invoice?

**Pull request merged**